pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -130,9 +136,10 @@ private void initializeApplicationQpsQuotas() {
 
       String appName = entry.getKey();
       double appQpsQuota =
-          entry.getValue() != null && entry.getValue() != -1.0d ? entry.getValue() : _defaultQpsQuotaForApplication;
+          entry.getValue() != null && entry.getValue() >= MIN_APP_QUOTA ? entry.getValue()
+              : _defaultQpsQuotaForApplication;
 
-      if (appQpsQuota < 0) {
+      if (appQpsQuota < MIN_APP_QUOTA) {
```

Comment:
Same for other places
```suggestion
      if (appQpsQuota <= 0) {
```

Reply:
See comment above.